packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -900,10 +911,6 @@ export const createAzureCommunicationCallAdapterFromClient = async (
   locator: CallAdapterLocator
 ): Promise<CallAdapter> => {
   const deviceManager = (await callClient.getDeviceManager()) as StatefulDeviceManager;
-  /* @conditional-compile-remove(unsupported-browser) */
-  const feature = callClient.feature(Features.DebugInfo);
-  /* @conditional-compile-remove(unsupported-browser) */
-  await feature.getEnvironmentInfo();
```

Comment:
Question that could be a bad idea: should we do this anyway to prime the call in the call composite?
Concern is that this is asynchronous:
![image](https://user-images.githubusercontent.com/2684369/200398952-ab194e5d-b37a-455a-8567-751a9496a9bd.png)
meaning we could flash the configuration page if its not primed

Reply:
Problem is if this is primed the adapter will set the page to `unsupportedEnvironement` and when the featrue is opted out you land on a white screen. I don't see value in adding a `clearEnvironmentInfo()` handler to use here, do you think that we should?

Reply:
Hmm, this is a problem. If someone calls getEnvironmentInfo() it will have a sudden side effect that the whole composite will go blank then

Reply:
I will look into making sure the composite is paying attention. I think if Contoso uses this function, they are in a sense opting into the feature. I will make sure the comment for this on the interface acknowledges this.

Reply:
> if Contoso uses this function, they are in a sense opting into the feature

If they have explicitly opted out somewhere in our API (in the composite interface), calling a function should never "opt them back in", particularly when the names don't align (nothing in name of `adapter.getEnvironmentInfo` suggest it'll override my opt-out property on a different interface)